.eslintrc

```diff
@@ -13,6 +13,7 @@
     "jest": true,
```

Comment:
Oh, huh. I must have missed this in the original patch. What about moving all of these test-specific globals to [tests/.eslintrc](https://github.com/mozilla/addons-frontend/blob/master/tests/.eslintrc)?